Improve "About Governance WG" page on the website (#384)
cncf/tag-contributor-strategy: website/content/about/working-groups/governance.md

url: /about/governance/
---

{{% blocks/lead color="primary" align="left" %}}

Want to implement good open source governance? We can assist you with best practices, resources, and guidance on how!

Read our [Charter](https://github.com/cncf/tag-contributor-strategy/blob/main/governance/README.md)

{{% /blocks/lead %}}

<div class="section-group">

{{% blocks/section color="dark" %}}

> The Governance Growth Working Group aim to assist all CNCF projects with implementing good open source governance practices.

Reviewer: ```suggestion
The Governance Working Group assists CNCF projects with implementing good open source governance practices.
```

Author: Oops, now you know I copy pasted stuff from the other pages I prepared 🤣

> We aim to provide best practices and resources for managing and interacting with contributors to CNCF projects.

Reviewer: ```suggestion
We provide best practices and resources for managing and interacting with contributors to CNCF projects.
```

{{% /blocks/section %}}

</div>

<div class="section-group">

{{% blocks/section color="secondary" %}}

## Goals

<div class="text-left">

* Create, share, and maintain documentation on implementing good open source governance for CNCF projects.
* Coach and advise CNCF projects, and prospective projects, leaders in implementing and improving governance.
* Advise the CNCF TOC on updating governance criteria for acceptance and graduation.

</div>

{{% /blocks/section %}}

</div>

<div class="section-group">
{{< blocks/section color="primary" >}}

## How your community can benefit

<div class="text-left">

* Have a look at our [Governance Guide](/maintainers/governance/) to see best practices, get advice and have inspiration while setting up or improving your project and community governance

> * Copy and modify one of our codified [Governance Templates](http://localhost:1313/maintainers/templates/governance-intro/) depending on your community and project

Reviewer: ```suggestion
* Copy and modify one of our codified [Governance Templates](/maintainers/templates/governance-intro/) depending on your community and project
```

> * Get assistance by reach out to us to any governance related issues

Reviewer: ```suggestion
* Get assistance on governance related issues
```

</div>

{{% /blocks/section %}}
</div>

<div class="section-group">
{{< blocks/section color="primary" >}}

## How you can contribute

> <div class="text-left">

Reviewer: I can't figure out why the text for this section is showing up on the same line as the heading. I'm going to merge and we can fix that visual quirk later though.
